src/spec/context.cr

```diff
@@ -7,6 +7,11 @@ module Spec
   abstract class Context
     # All the children, which can be `describe`/`context` or `it`
     getter children = [] of NestedContext | Example
+
+    def randomize(randomizer = Random::DEFAULT)
```

Comment:
There's no need for a default value here because it's never called without a value.

Reply:
I had original had that for testing purposes, but backed it out, so yeah I'll remove